import.sh

```diff
@@ -1,4 +1,32 @@
 #!/usr/bin/env sh
+#
+# This script is meant to be used to update the version of rocksdb in this repo.
+#
+# To use it, update the version number in the URL to the tar.gz file below, then
+# run it. If you're lucky, everything will just work, but you may get an error
+# about a patch not applying cleanly. If that's the case, try to fix up the
+# patch file so that it will apply cleanly, then reset the rest of the
+# repository and try again.
+#
+# Once things have run cleanly, you should have a fairly large diff of the
+# actual rocksdb code, potentially along with some added or removed symlink
+# files in the base of the repository.
+#
+# Once you've successfully run this script, you should also manually verify that
+# the recommended build flags for darwin and linux mostly match up with what's
+# specified in the cgo_flags.go file in the base of the repo.
+# To do so go through the below process on both a mac machine and a linux
+# machine (or a linux docker container):
+#   cd internal
+#   ./build_tools/build_detect_platform tmp.txt
+#   # Manually compare the PLATFORM_CCFLAGS and PLATFORM_CXXFLAGS to what's in
+#   # cgo_flags.go
+#
+# Ask @tamird if you run into issues along the way.
+#
+# After committing locally and before trying to build/test cockroachdb, you
+# should run the command below to ensure your repo is in a clean state:
+#   git clean -dxf
```

Comment:
Yeah. I added a small note that you should start from a clean repo, but I don't expect anyone to try bumping the version without starting from a clean repo.